pants

```diff
@@ -1,84 +1,264 @@
 #!/usr/bin/env bash
-# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Is this a copy paste of our `aiq/build-support/pants/versions/1.27.0/pants`?

Reply:
yes